client/my-sites/domains/domain-management/dns/restore-default-cname-record-dialog.tsx

```diff
@@ -25,14 +36,14 @@ function RestoreDefaultCnameRecordDialog( { onClose, visible } ) {
 		},
 	];
 
+	const targetPlatformMessage = domain?.isGravatarDomain
+		? __( 'Restoring the record will point this domain to your Gravatar profile.' )
+		: __( 'Restoring the record will point this domain to your WordPress.com site.' );
```

Comment:
```suggestion
		? __( 'Restoring the record will point the www subdomain to your Gravatar profile.' )
		: __( 'Restoring the record will point the www subdomain to your WordPress.com site.' );
```
Maybe this is a little more precise?